External/imtool3D_td/External/bids-matlab/+bids/+util/jsonencode.m

```diff
@@ -54,6 +54,11 @@
         fclose(fid);
     end
     varargout = { txt };
+
+elseif exist('savejson','file') == 2   
+    disp('this');                 % JSONlab
```

Comment:
Is this line a leftover debugging print?

Reply:
Yeap, will remove it.